.github/workflows/ai-lab-e2e-nightly-windows.yaml

```diff
@@ -200,6 +200,28 @@ jobs:
         # check logs
         podman logs -f pde2e-podman-run
 
+    - name: Build Podman Desktop Electron Inspect Enabled binary
+      run: |
+        podman run --rm -d --name pde2e-builder-run \
+          -e TARGET_HOST=$(cat host) \
+          -e TARGET_HOST_USERNAME=$(cat username) \
+          -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+          -e TARGET_FOLDER=pd-e2e \
+          -e TARGET_CLEANUP=false \
+          -e TARGET_RESULTS=results \
+          -e OUTPUT_FOLDER=/data \
+          -e DEBUG=true \
+          -v $PWD:/data:z \
+          quay.io/odockal/pde2e-builder:${{ env.PDE2E_IMAGE_VERSION }} \
```

Comment:
question: is is expected to have `quay.io/odockal` there ?

Reply:
yes it is. Although, not right. I need to move the images to the official organization. I have created an issue: https://github.com/containers/podman-desktop-e2e/issues/268